x-pack/plugin/security/build.gradle

```diff
@@ -137,6 +137,11 @@ artifacts {
     archives jar
     testArtifacts testJar
 }
+
+sourceSets.main.resources {
+    srcDir 'src/main/config'
```

Comment:
Maybe place this in a subdir under resources, so we don't pollute the top level?